src/backends/gstreamer/audio_sink.rs

```diff
@@ -81,19 +55,16 @@ impl AudioSink for GStreamerAudioSink {
                     .mul_div_floor(gst::SECOND_VAL, rate as u64)
                     .unwrap()
                     .into();
-                let next_pts: gst::ClockTime = (sample_offset + n_samples)
+                let next_pts: gst::ClockTime = (sample_offset + WEBAUDIO_RENDER_QUANTUM)
                     .mul_div_floor(gst::SECOND_VAL, rate as u64)
                     .unwrap()
                     .into();
                 buffer.set_pts(pts);
                 buffer.set_duration(next_pts - pts);
                 let mut map = buffer.map_writable().unwrap();
```

Comment:
... and here the read-only memory would be copied into writeable memory so you can modify it.

It would be more efficient to create a buffer with writeable memory to begin with. `with_size` as before would do that, you only have to fill it with silence then e.g. with [this](https://sdroege.github.io/rustdoc/gstreamer/gstreamer_audio/struct.AudioFormatInfo.html#method.fill_silence)